### PR TITLE
Add migration checks

### PR DESCRIPTION
**Pull request by aronerben**

### CODE REVIEW

**Comment by aronerben** [comment]
@jerben I decided to leave out the progress bar for now, since it isn't too high priority. What do you think?

**Comment by aronerben** [comment]
https://github.com/oxidizing/sihl/pull/367

**Comment by jerben** [comment]
I agree, let's focus on more urgent stuff for now but this would be nice to have at some point.

File: sihl-persistence/src/dune [hunk 1]
```diff
@@ -1,4 +1,6 @@
 (library
  (name sihl_persistence)
  (public_name sihl-persistence)
- (libraries sihl-facade))
+ (libraries sihl-facade)
```
Comment (jerben) [inline sihl-persistence/src/dune:4]: @aronerben Please don't use ppx_fields_conv, I started to remove it in Sihl and instead use record access like `record.Some.Module.field`

**Review by jerben — CHANGES_REQUESTED** [review]

**Comment by jerben** [comment]
Looks good, once the ppx is removed it can be merged imo, thanks!

File: sihl-persistence/src/dune (hunk 1 above)
Comment (aronerben) [reply sihl-persistence/src/dune]: @jerben Thanks, done.

**Pull request merged**